src/phpDocumentor/Configuration/ValueObjects/VersionObject.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @link https://phpdoc.org
+ */
+
+namespace phpDocumentor\Configuration\ValueObjects;
+
+use phpDocumentor\Dsn;
+use phpDocumentor\Path;
+
+class VersionObject
+{
+    /** @var string */
+    public $number;
```

Comment:
I prefer getters for all properties in this class. In the end, this object should be immutable. The only location where mutations are allowed is in the `Configuration` namespace.

Reply:
Getters were added, but I had difficulties for setters because subsets of the api property are overwritten dozens times which make the process really heavy. You have to make a copy of the whole array, changing what need to be changed locally and then push the changes.

It may be easier once we have more Specification objects because we'll be able to write a subset via a sub-object.